samples/src/Microsoft.Azure.IIoT.App/src/Shared/WorkbookLink.razor

```diff
@@ -0,0 +1,56 @@
+﻿@*------------------------------------------------------------
+  Copyright (c) Microsoft Corporation.  All rights reserved.
+  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+ ------------------------------------------------------------*@
+
+@using Microsoft.Azure.IIoT.App.Runtime
+@using Microsoft.AspNetCore.Components
+@using System
+
+@inject NavigationManager NavigationManager
+@inject Config Configuration
+
+<AuthorizeView>
+    <Authorized>
+        @if(!String.IsNullOrEmpty(_workbookLink))
+        {
+            <a href="@_workbookLink" target="_blank">Go to IIoT Workbook</a>
+        }
+    </Authorized>
+    <NotAuthorized>
+    </NotAuthorized>
+</AuthorizeView>
+
+@code {
+    private string _workbookLink { get; set; } = null;
+
+
+    /// <summary>
+    /// OnInitialized
+    /// </summary>
+    protected override void OnInitialized()
+    {
+        CreateWorkbookLink();
+    }
+
+    /// <summary>
+    /// Create workbook link
+    /// </summary>
+    private void CreateWorkbookLink()
+    {
+        if (!String.IsNullOrEmpty(Configuration.WorkbookId))
+        {
+            var index = Configuration.AuthAudience.LastIndexOf('/');
+            if (index > 0)
+            {
+                _workbookLink = "https://portal.azure.com/#@" +
+                    Configuration.AuthAudience.Substring(8, index - 8) +
+                    "/resource/subscriptions/" + Configuration.SubscriptionId +
+                    "/resourceGroups" + Configuration.AuthAudience.Substring(index).Replace("-service", "") +
```

Comment:
There is no guarantee that the App Registration audience will have `https://<tenant-id>/<resource-group>-service` format. In the case of the Helm chart clients will be providing information about their own app registrations which does not have to match our format. I would suggest to either programmatically get the resource group or to use a PCS variable for that. Marc is already outputting `PCS_RESOURCE_GROUP` to `.env` file. I would suggest pushing it to KeyVault and using it.

Reply:
Add PCS_RESOURCE_GROUP to Vault. Now I use this one